library/pom.xml

```diff
@@ -315,7 +315,17 @@
       <artifactId>joda-time</artifactId>
       <version>2.9.1</version>
     </dependency>
-
+    <dependency>
```

Comment:
As discussed on dev@, we don't want to add dependencies to malhar-library that aren't broadly applicable.